tools/src/main/resources/upgrade/updatedatabase_1_6_x.bat

```diff
@@ -13,7 +13,7 @@
 @REM  Now copy the data from openmrs_backup into the openmrs_script
     
 
-@mysql --port=3306 -u root openmrs < upgrade.sql
+@mysql --port=3306 -u root openmrs < upgrade_1_6_x.sql
```

Comment:
Remove the @ at the beginning of this line so that the logging that @carapai has enabled can be seen